config.json

```diff
@@ -463,7 +463,8 @@
         "functions",
         "mathematics"
       ],
-      "uuid": "1f68f837-6432-4f9e-afae-0cb17f8e0153"
+      "unlocked_by": "triangle",
```

Comment:
This was missing an `unlocked_by`.  If anyone has a better exercise to unlock this one, let me know.

Reply:
This looks good to me

Reply:
`triangle` looks good, but maybe `pascals-triangle` might be better (which also unlocks `triangle`). Going by some of the other tracks it seems to be the case that a lot only use a single level of unlock depth. i.e. one core exercise unlocks multiple non-core exercises rather than one core exercise unlocks one non-core which unlocks another non-core, etc.

Reply:
Oh, good catch. Yes, I think that although it is possible to use more than one level of unlocking, it's not recommended.